src/elements/play-project-button.ts

```diff
@@ -0,0 +1,187 @@
+      font-weight: 600;
+      line-height: 20px;
+      letter-spacing: -0.3px;
+    }
+
+    .project-pen {
+      display: flex;
+      flex-direction: row;
+    }
+
+    :host([size='small']) .project-pen {
+      column-gap: 8px;
+      padding-top: 8px;
+      padding-right: 8px;
+      padding-bottom: 8px;
+      padding-left: 12px;
+      border-top-left-radius: 16px;
+      border-bottom-left-radius: 16px;
+    }
+
+    :host([size='medium']) .project-pen {
+      column-gap: 8px;
+      padding-top: 10px;
+      padding-right: 12px;
+      padding-bottom: 10px;
+      padding-left: 16px;
+      border-top-left-radius: 20px;
+      border-bottom-left-radius: 20px;
+    }
+
+    :host([size='small']) .new-from-template {
+      padding-top: 8px;
+      padding-right: 8px;
+      padding-bottom: 8px;
+      padding-left: 8px;
+      border-top-right-radius: 16px;
+      border-bottom-right-radius: 16px;
+    }
+
+    :host([size='medium']) .project-options {
+      padding-top: 10px;
+      padding-right: 10px;
+      padding-bottom: 10px;
+      padding-left: 8px;
+      border-top-right-radius: 20px;
+      border-bottom-right-radius: 20px;
+    }
+
+    .project-options:hover,
+    .project-template:hover {
+      background-color: var(--color-secondary-background-hovered);
+    }
+
+    .project-pen:active,
+    .project-options:active {
+      background-color: var(--color-secondary-background-active);
+    }
+
+    .project-pen:focus,
+    .project-options:focus {
+      outline-color: var(--color-brand-background);
+    }
+
+    .divider {
+      width: 1px;
+      background-color: var(--color-secondary-background-decor);
+    }
+
+    :host([size='small']) .divider {
+      height: 16px;
+    }
+
+    :host([size='medium']) .divider {
+      height: 20px;
+    }
+  `
+
+  @property({attribute: false}) srcByLabel?: {readonly [key: string]: string}
+  @property() size: SizeOptions = 'medium'
+
+  protected override render(): TemplateResult {
+    return html`
+      <div class="container">
+        <button
+          class="project-pen"
+          @click=${() =>
+            this.dispatchEvent(
+              Bubble<string>('edit-src', this.srcByLabel?.Default || '')
+            )}
+          title="Project Pen"
+        >
+          <play-icon
+            size=${iconSizes[this.size]}
+            icon="download-outline"
+          ></play-icon>
+          <span>Project</span>
+        </button>
+        <div class="divider"></div>
+        <play-dropdown-menu direction="down">
+          <div slot="trigger">
+            <button class="project-options" title="Project Options">
+              <play-icon
+                size=${iconSizes[this.size]}
+                icon="caret-down-outline"
+              ></play-icon>
+            </button>
+          </div>
+          <div slot="menu">
+            <play-list-item
+              label="Save"
+              icon="download-outline"
+            ></play-list-item>
```

Comment:
we should have "Save", "Load", and "Export"

and maybe have Save and Load commented out for now until we wire them up? but the code can go ahead and be here :)